test/ipynb/results.py

```diff
@@ -146,16 +147,16 @@ def summary(self):
         ret = ""
 
         if len(self.mismatch) >= 2:
-            Results._zipfiles(self.mismatch, "mpl/mismatch.zip")
+            Results._zipfiles(self.mismatch, "mpl/Circuit/mismatch.zip")
```

Comment:
Folder names should be lowercase `circuit` and `graph` please 😊

Reply:
done